src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -191,6 +191,9 @@
 
     private static Map<String, Supplier<SSTableFormat<?, ?>>> sstableFormatFactories;
 
+    /** The sstable compression options */
```

Comment:
The usefulness of this comment is questionable. It is pretty obvious this is about SSTable compression from the name of that variable itself.